satpy/etc/readers/aws1_mwr_l1b_nc.yaml

```diff
@@ -0,0 +1,535 @@
+        standard_name: satellite_zenith_angle
+        horn: "2"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_zenith_horn3:
+        name: satellite_zenith_horn3
+        file_type: aws_l1b_nc
+        file_key: data/navigation/aws_satellite_zenith_angle
+        standard_name: satellite_zenith_angle
+        horn: "3"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_zenith_horn4:
+        name: satellite_zenith_horn4
+        file_type: aws_l1b_nc
+        file_key: data/navigation/aws_satellite_zenith_angle
+        standard_name: satellite_zenith_angle
+        horn: "4"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_azimuth_horn1:
+        name: satellite_azimuth_horn1
+        file_type: aws_l1b_nc
+        file_key: data/navigation/aws_satellite_azimuth_angle
+        standard_name: satellite_azimuth_angle
+        horn: "1"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_azimuth_horn2:
+        name: satellite_azimuth_horn2
+        file_type: aws_l1b_nc
+        file_key: data/navigation/aws_satellite_azimuth_angle
+        standard_name: satellite_azimuth_angle
+        horn: "2"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_azimuth_horn3:
+        name: satellite_azimuth_horn3
+        file_type: aws_l1b_nc
+        file_key: data/navigation/aws_satellite_azimuth_angle
+        standard_name: satellite_azimuth_angle
+        horn: "3"
+        coordinates:
+            - longitude
+            - latitude
+
+    satellite_azimuth_horn4:
+        name: satellite_azimuth_horn4
+        file_type: aws_l1b_nc
+        file_key: data/navigation/aws_satellite_azimuth_angle
+        standard_name: satellite_azimuth_angle
+        horn: "4"
+        coordinates:
+            - longitude
+            - latitude
+
+file_types:
+    aws_l1b_nc:
+        # W_XX-OHB-Unknown,SAT,1-AWS-1B-RAD_C_OHB_20230707124607_G_D_20220621090100_20220621090618_T_B____.nc
+        # W_XX-OHB-Stockholm,SAT,AWS1-MWR-1B-RAD_C_OHB_20230823161321_G_D_20240115111111_20240115125434_T_B____.nc
+        # W_XX-OHB-Stockholm,SAT,AWS1-MWR-1B-RAD_C_OHB_20230816120142_G_D_20240115111111_20240115125434_T_B____radsim.nc
+        file_reader: !!python/name:satpy.readers.mwr_l1b.AWS_EPS_Sterna_MWR_L1BFile
+        file_patterns: [
+        'W_XX-OHB-Stockholm,SAT,{platform_name}-MWR-1B-RAD_C_OHB_{processing_time:%Y%m%d%H%M%S}_G_D_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_{disposition_mode:1s}_{processing_mode:1s}____.nc',
+        'W_XX-OHB-Stockholm,SAT,{platform_name}-MWR-1B-RAD_C_OHB__{processing_time:%Y%m%d%H%M%S}_G_D_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_T_B____.nc',
+        'W_XX-OHB-Stockholm,SAT,{platform_name}-MWR-1B-RAD_C_OHB_{processing_time:%Y%m%d%H%M%S}_G_D_{start_time:%Y%m%d%H%M%S}_{end_time:%Y%m%d%H%M%S}_T_B____radsim.nc']
```

Comment:
The patterns should have "tags" in place of `OHB` (two places) and `Stockholm.

Reply:
Ahh, yes, will fix